src/wp-admin/about.php

```diff
@@ -50,187 +42,168 @@
 					?>
 				</p>
 			</div>
-			<div class="column has-border" style="background-color:#d1d1e4;background-color:var(--global--color-purple)">
-				<h3><?php _e( 'Built-in patterns' ); ?></h3>
-				<p><?php _e( 'If you’ve not had the chance to play with block patterns yet, all default themes now feature a range of block patterns that let you master complex layouts with minimal effort. Customize the patterns to your liking with the copy, images and colors that fit your story or brand.' ); ?></p>
+			<div class="column about__image">
+				<img src="data:image/svg+xml,%3Csvg width='269' height='120' xmlns='http://www.w3.org/2000/svg'%3E%3Crect x='0' y='0' width='100%25' height='100%25' fill='%23ddd' /%3E%3Ctext text-anchor='middle' font-family='sans-serif' font-size='24' y='50%25' x='50%25'%3EImage%3C/text%3E%3C/svg%3E" alt="" />
 			</div>
 		</div>
 
 		<hr />
 
-		<div class="about__section has-2-columns">
-			<h2 class="is-section-header"><?php _e( 'For developers' ); ?></h2>
+		<div class="about__section has-subtle-background-color">
 			<div class="column">
-				<h3><?php _e( 'REST API authentication with Application Passwords' ); ?></h3>
-				<p><?php _e( 'Thanks to the API’s new Application Passwords authorization feature, third-party apps can connect to your site seamlessly and securely. This new REST API feature lets you see what apps are connecting to your site and control what they do.' ); ?></p>
+				<h2 class="is-smaller-heading"><?php _e( 'A Simpler Default Color Palette' ); ?></h2>
 			</div>
+		</div>
+
+		<div class="about__section has-subtle-background-color">
+			<div class="column about__image">
+				<div class="about__image-comparison">
+					<div class="about__image-comparison-resize">
+						<img src="https://ryelle.codes/core/colors/screenshots-branch/ss-0.png" />
+					</div>
+					<img src="https://ryelle.codes/core/colors/screenshots/ss-0.png" />
+				</div>
+			</div>
+		</div>
+
+		<div class="about__section has-2-columns has-subtle-background-color">
 			<div class="column">
-				<h3><?php _e( 'More PHP 8 support' ); ?></h3>
 				<p>
 					<?php
 					printf(
-						/* translators: %s: WordPress and PHP 8 dev note link. */
-						__( '5.6 marks the first steps toward WordPress Core support for PHP 8. Now is a great time to start planning how your WordPress products, services and sites can support the latest PHP version. For more information about what to expect next, <a href="%s">read the PHP 8 developer note</a>.' ),
-						'https://make.wordpress.org/core/2020/11/23/wordpress-and-php-8-0/'
+						/* translators: 1: WCAG information link. */
+						__( 'This new streamlined color palette collapses all the colors that used to be in the WordPress source code down to seven core colors and a range of 56 shades that meet the <a href="%1$s">WCAG 2.0 AA recommended contrast ratio</a> against white or black.' ),
```

Comment:
```suggestion
						__( 'This new streamlined color palette collapses all the colors that used to be in the WordPress source code down to seven core colors and a range of 56 shades that meet the <a href="%1$s" target="_blank">WCAG 2.0 AA recommended contrast ratio</a> against white or black.' ),
```

Reply:
I thought it was generally bad accessibility practice to force links to open in a new window?

Reply:
Hmm, I'll check the handbook to see what the a11y team recommends. I usually open in new tab if it leaves the current site to prevent the feeling of losing your spot.

Reply:
Let's disregard this suggestion - based on our handbook, if we do not tell them we are opening the link in a new tab,  we should instead let the user decide what to do. 
https://make.wordpress.org/accessibility/handbook/content/link-destinations/